data/tests/test_extraction_open_data.py

```diff
@@ -11,14 +13,27 @@ def test_extraction_teledeclaration(self):
         schema_cols = [i["name"] for i in schema["fields"]]
         canteen = CanteenFactory.create()
         applicant = UserFactory.create()
+
+        assert len(_extract_dataset_teledeclaration(year=1990)) == 0, "There should be no teledeclaration"
+
         for year_td in [2021, 2022]:
             diagnostic = DiagnosticFactory.create(canteen=canteen, year=year_td, diagnostic_type=None)
-            Teledeclaration.create_from_diagnostic(diagnostic, applicant)
+            teledeclaration = Teledeclaration.create_from_diagnostic(diagnostic, applicant)
```

Comment:
Généralement pas idéale l'assignation dans le `for` (pas bloquant pour le merge)

Reply:
Oui, en plus on a failli se faire avoir avec